tests/cpp/test_multidevice_transformer.cpp

```diff
@@ -26,21 +28,26 @@
 
 namespace nvfuser {
 
+namespace {
```

Comment:
constexpr are internal linkage by default, so I'd remove the anonymous namespace.